commands/command_test.go

```diff
@@ -0,0 +1,101 @@
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlUnitTestSuite) TestCommandListCmdF() {
+
+	s.Run("List all commands from all teams", func() {
+		printer.Clean()
+		team1ID := "team-id-1"
+		team2Id := "team-id-2"
+
+		commandTeam1ID := "command-team1-id"
+		commandTeam2Id := "command-team2-id"
+		teams := []*model.Team{
+			&model.Team{Id: team1ID},
+			&model.Team{Id: team2Id},
+		}
+
+		team1Commands := []*model.Command{
+			&model.Command{
+				Id: commandTeam1ID,
+			},
+		}
+		team2Commands := []*model.Command{
+			&model.Command{
+				Id: commandTeam2Id,
+			},
+		}
+
+		cmd := &cobra.Command{}
+		s.client.EXPECT().GetAllTeams("", 0, 10000).Return(teams, &model.Response{Error: nil}).Times(1)
+		s.client.EXPECT().ListCommands(team1ID, true).Return(team1Commands, &model.Response{Error: nil}).Times(1)
+		s.client.EXPECT().ListCommands(team2Id, true).Return(team2Commands, &model.Response{Error: nil}).Times(1)
+		err := listCommandCmdF(s.client, cmd, []string{})
+		s.Len(printer.GetLines(), 2)
+		s.Equal(team1Commands[0], printer.GetLines()[0])
+		s.Equal(team2Commands[0], printer.GetLines()[1])
+		s.Nil(err)
```

Comment:
nit: I'd push up this check to be the first one and modify it to be a required, as having an error here will invalidate the other checks and may cause an index out of range check

```suggestion
		s.Require().Nil(err)
```